README.md

```diff
@@ -60,12 +60,13 @@ module "example" {
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
 | account\_ids | AWS account IDs that are allowed to assume the role that allows read-only access to the specified Terraform state. | `list(string)` | `[]` | no |
+| additional\_role\_tags | Tags to apply to the IAM role that allows read-only access to the specified Terraform state, in addition to the provider's default tags. | `map(string)` | `{}` | no |
 | assume\_role\_policy\_description | The description to associate with the IAM policy that allows assumption of the role that allows read-only access to the specified Terraform state.  Note that the first "%s" in this value will get replaced with the role\_name variable and the second "%s" will get replaced with the terraform\_account\_name variable. | `string` | `"Allow assumption of the %s role in the %s account."` | no |
 | assume\_role\_policy\_name | The name to assign the IAM policy that allows assumption of the role that allows read-only access to the specified Terraform state.  Note that the "%s" in this value will get replaced with the role\_name variable. | `string` | `"Assume%s"` | no |
+| aws\_region | The AWS region where the non-global resources are to be provisioned (e.g. "us-east-1"). | `string` | `"us-east-1"` | no |
```

Comment:
This snuck in but I don't believe it belongs.

Reply:
Great catch!  See commit f0e8ca1.